app/models/phase.rb

```diff
@@ -29,7 +29,7 @@ class Phase < ApplicationRecord
   # has_one :winner, class_name: 'PhaseWinner'
   has_many :evaluator_invitations, dependent: :destroy
   has_many :challenge_phases_evaluators, dependent: :destroy
-  has_many :evaluators, through: :challenge_phases_evaluators, source: :user
+  has_many :evaluators, -> { where(role: "evaluator") }, through: :challenge_phases_evaluators, source: :user
```

Comment:
there is an issue here if I add a non-evaluator role user to the phase, they will not be shown in the list of evaluators. for example, I invited a user who is currently a solver to be an evaluator but they don't show up in the list below:
![Screenshot 2025-02-04 at 10 18 54 AM](https://github.com/user-attachments/assets/230c107d-1b79-49d8-af3b-6cc5bd109e8a)
for #75 we just want to filter the list of available evaluators so that users can't be assigned to submissions until their role is updated to `evaluator`. (this list)
![Screenshot 2025-02-03 at 7 49 14 AM](https://github.com/user-attachments/assets/560b5540-d624-4cf3-bf4b-e8f341d983bf)